roles/openshift_node/tasks/upgrade/bootstrap_changes.yml

```diff
@@ -74,6 +75,11 @@
     mode: 0600
   when: inventory_hostname in groups.oo_masters_to_config
 
+- name: Check for client-ca.crt
```

Comment:
This seems wrong - if bootstrap is true, this file should always exist once the node has started at least once, so we always want to create the symlink.